PowerControl/Controller.cs

```diff
@@ -50,6 +55,21 @@ public Controller()
             if (Instance.WantsRunOnStartup)
                 startupManager.Startup = true;
 
+            DeviceManager.LoadDisplays();
+            contextTimer = new System.Windows.Forms.Timer();
```

Comment:
Can we isolate this? I also hope that this takes exactly one iteration, otherwise we might have infinite loop :)

```csharp
var contextTimer = new System.Windows.Forms.Timer();
contextTimer.Tick += (_, _) =>
{
    context = SynchronizationContext.Current;
    contextTimer.Stop();
};
contextTimer.Start();
```

Reply:
Isolate by moving it to other class or to other method? Yeah, it clicks on first tick :)

Reply:
To be sure I will change interval from 100 to 200

Reply:
I've tried, but idk if that's what you mean

Reply:
I could've move the whole contextTimer set up to the method